apps/core/views.py

```diff
@@ -79,19 +79,24 @@ def import_group(request):
     return render(request, 'import_group.html', data)
 
 
+def get_xml(jobT):
+    """Takes JobTemplate object and returns its prettified job XML"""
+    jobG = JobGen()
+    xml = jobG.getXML(jobT, reserve=True)
+    try:
+        soup = BeautifulSoup(xml, "xml")
+    except:
+        raise Exception("Bad xml format or something is bad")
```

Comment:
I'd strongly suggest fixing this. This block catches *all* exceptions. You have *no* idea what's being caught, it could be anything from a keyboard event to a database lock-up to a file not found exception to god knows what. And raising a completely generic `Exception` perpetuates the same bad logic to other parts of code.